tests/Oryx.BuildImage.Tests/BenvTest.cs

```diff
@@ -108,7 +122,7 @@ public BenvTest(ITestOutputHelper output)
                 result.GetDebugInfo());
         }
 
-        [Theory]
+        [Theory, Trait("category", "vso-focal")]
         [InlineData("dotnet", "vso-focal")]
```

Comment:
nit: there appears to be a few test where the `debianImageFlavor` is provided as an argument, but is the same for all of the tests in the suite -- would you be able to clean up these tests so we don't need to duplicate the argument and just have it hard-coded into the tests?

Reply:
Yes, I recently pushed out an update to remove `debianImageFlavor` from tests that don't need it